modules/auxiliary/gather/zoomeye_search.rb

```diff
@@ -51,6 +51,13 @@ def initialize(info = {})
         OptBool.new('DATABASE', [false, 'Add search results to the database', false])
       ]
     )
+
+    register_advanced_options(
+      [
+        OptString.new('UserAgent', [false, 'The User-Agent header to use for all requests', 'Wget/1.21.2 (linux-gnu)' ])
```

Comment:
Does the API only support wget? I believe if these options are removed, the user agent will be randomized - would that be an issue here?

Reply:
I don't know, I copied that from the shodan module.
The API isn't free anymore and requires a monthly fee to be used so I just can't test it.
I did not know user agent is randomized when not specified.

Reply:
I have checked the api references, and there's no more changes than those I listed, I had written this module a few years ago and it was working, so I think this will work.